### ISSUE TITLE
Error Running cnn_mnist_experiments.m

### ISSUE THREAD

**Issue by travesty82** [vlfeat/matconvnet#399]
I recently downloaded and compiled MatConvNet onto my computer (2/1/2016). I tried to run the MNIST example experiment, and I get these errors:

    Error using **vl_argparse** (line 119)
    Unknown parameter 'useBnorm'

    Error in **cnn_mnist **(line 19)
    opts = vl_argparse(opts, varargin) ;

    Error in **cnn_mnist_experiments** (line 3)
    [net_bn, info_bn] = cnn_mnist(...
 
I have not made any alterations to the code, but literally tried to run this after going through the basic quick install and CPU compilation. Please help, as I would like to use this platform for research. Thanks.

**Comment by travesty82** [comment]
This error is on the latest MatConvNet beta 18 version, just to be clear. I did everything on the Quick Start page, and the Installing page (for CPU purposes). Please help as I'm trying to use this for research, and it came highly recommended. Thanks.

**Comment by vedaldi** [comment]
Hi, thanks. You can

* Use the `cnn_mnist` example instead. The script you are running is supposed to run a bunch of training sessions and compare the results.

* Correct the bug by replacing `'batchNorm` with `'batchNormalization'`

**Comment by travesty82** [comment]
Okay. I tried running the cnn_mnist.m as is, but still have issues. When you say replace 'batchNorm' with 'batchNormalization', I don't see where. 'batchNorm' doesn't show up in the cnn_mnist.m code, it's already set to 'batchNormalization'. I copied this line
`[net_bn, info_bn] = cnn_mnist('expDir', 'data/mnist-bnorm', 'useBnorm', true);`

and ran it, and got the same error about 'useBnorm' being unknown. I didn't know what to leave out or keep as input variables.

I was able to run it by leaving out the **'useBnorm', true** portion of the function inputs. My question is, should 'useBnorm' even be there or was the command updated to something else? Otherwise it's always saying that command is unknown. I'm sure it is there, or some variant, but I'd like to use all of the options of this platform. Thanks again.
